pages/curriculum.tsx

```diff
@@ -130,13 +132,17 @@ export const Curriculum: React.FC<Props> = ({ lessons, alerts }) => {
   }, [])
   useEffect(() => {
     if (data && data.session) {
+      if (data.session.user && data.session.user.isConnectedToDiscord) {
```

Comment:
Is there a big benefit to adding the extra `isConnectedToDiscord` field vs just checking that the discord username string isn't empty?

Reply:
ah, yeah. the discord username & avatar stuff is not available to session query, only userInfo query. 

the tokens and expiry are available in the req.user object, so I had to make use of those without exposing them to the frontend.

Reply:
Ok that makes sense